File: programmers/Python/Lv1/완주하지 못한 선수.py
def solution(participant, completion):
    
    participant_dict={}

    for i in participant:
        participant_dict[i]=participant_dict.get(i,0)+1

    for i in completion:
        if i in participant_dict:
            participant_dict[i]-=1
            if participant_dict[i]==0:
                del participant_dict[i]

    return ''.join(participant_dict.keys())

File: programmers/C++/Lv1/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";

    unordered_map<string, int> pp;

    for (string name : participant){
      pp[name]++;
    }

    for (string name : completion){
      pp[name]--;
    }

    for (pair<const string, int> p : pp){
      if (p.second==1){
        answer = p.first;
      }
    }

    return answer;
}